### ISSUE TITLE
USER ISSUE: [7.6.0] Allocating crashes the game

### ISSUE THREAD

**Issue by BlazeGuy2** [StrangeLoopGames/EcoIssues#8354]
**Version:** 0.7.6.0 beta staging-f5159538

**Steps to Reproduce:**
1. go to treasury made from /spawneconomy (also repro'ed from /testtreasury)
2. allocate

**Expected behavior:**
Allocate the currency

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: Exception
Message:Original type is Object. Cannot convert from Object to Guid
Source:Eco.Shared

System.Exception: Original type is Object. Cannot convert from Object to Guid
   at Eco.Shared.Serialization.BSONValue.get_GuidValue()
   at Eco.Gameplay.Economy.Currency.<>c.<.cctor>b__45_0(BSONValue v)
   at Eco.Shared.Serialization.BsonManipulator.FromBson(BSONValue value, Type type)
   at Eco.Shared.Networking.RPCManager.UnpackArgs(INetClient client, BSONObject bson, RPCMethod[] methods, Object target, RPCMethod& targetMethod)</size>
```

**Comment by BlazeGuy2** [comment]
Not sure if this is also occurring on 7.5.1 or not

**Comment by BlazeGuy2** [comment]
Wait, eric fixed this #8211
